### DISCUSSION TITLE
如何解决上游应用的根目录路径问题

### DISCUSSION THREAD

**Discussion by 348465303** [apache/apisix#6400] [Q&A]
路由配置如下：
![4d057e238d63ed738142d9b5443930a](https://user-images.githubusercontent.com/17284334/154934765-dd44dd8b-93bc-44c3-8620-b8e2d54987fb.png)

现在代理一个mvc的项目，项目中的js、css的路径都是/xxxx.js，现在访问报404，我还要怎么配置才能解决这个问题

**Comment by starsz** [comment]
Hi @348465303.
Do you mean that
request "/test/zdkjr/xxxx.js" then proxy to xxxx.js to upstream?

**Comment by woyouyitouchenfeijv** [comment]
我们这么用的，js和css都可
![image](https://user-images.githubusercontent.com/27728236/155685338-28bc1a71-32f3-46ae-9e3c-23d95103888c.png)

**Reply by 348465303** [reply]
你的这个真实项目名是什么意思，我项目里的js和css路径都是/xxx.css这样的

**Reply by woyouyitouchenfeijv** [reply]
没有项目名的工程吗
No project name ？ 

可以试试设置一个域名，我在没项目名的时候。单独用域名去代理的访问
You can try setting up a domain name. When I don't have a project name. Use the domain name alone to proxy access

**Comment by wang-yeliang** [comment]
@348465303 @DengX1993 hi,Please communicate in English on public channels so that more people can participate in the discussion
